src/styles/editor.scss

```diff
@@ -60,11 +72,17 @@
   > #{$root}__content {
     background-color: $white-color;
     border: 1px solid $lighter-border-color;
-    border-bottom: 0;
-    border-top: 0;
     color: $black-color;
     flex-grow: 1;
     padding: 30px 20px;
+
+    &_border-bottom_none{
```

Comment:
Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/Leecason/element-tiptap/pullRequest?prid=5779312)

Reply:
I've fixed it